tests/TestCase.php

```diff
@@ -171,4 +171,18 @@ public function checkReportbackResponse($response)
         // Make sure we created a record in the reportback log table.
         $this->seeInDatabase('reportback_logs', ['reportback_id' => $response['data']['id']]);
     }
+
+    /**
+     * Create a reaction request.
+     *
+     * @return array
+     */
+    public function createReactionRequest($northstarId, $photoId, $postType)
+    {
+        return [
+            'northstar_id' => $northstarId,
+            'reactionable_id' => $photoId,
+            'reactionable_type' => $postType,
+        ];
+    }
 }
```

Comment:
I just realized I can probably also pull more into this helper function (sending the request and asserting the response). Will do that now!